CONTRIBUTING.md

```diff
@@ -174,34 +174,15 @@ Please copy and paste the following quotes for the languages:
 
 #### Environment Variable For Apple M1 Platform
 
-To set the Docker platform environment variable in your terminal tab, run:
+To set the following environment variable seems not be needed anymore, probably because Docker knows in its newest version on which CPU it runs and sets that to default.
```

Comment:
I am not a fan of guesswork in the documentation

Reply:
Done